### PR TITLE
Nat 2017/4/4 neighbourhoods with new features (require PIP)

### PR DESCRIPTION
**Pull request by nvkelso**
-  6912 records

### CODE REVIEW

**Comment by nvkelso** [comment]
Related: 

- https://github.com/whosonfirst-data/whosonfirst-data/pull/716
- https://github.com/whosonfirst-data/whosonfirst-data/pull/717

File: data/110/891/618/3/1108916183.geojson [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "lbl:latitude":-34.370212,
+    "lbl:longitude":150.896779,
+    "mz:is_funky":0,
+    "mz:is_hard_boundary":0,
+    "mz:is_landuse_aoi":0,
+    "mz:is_official":0,
+    "mz:max_zoom":18,
+    "mz:min_zoom":16,
+    "mz:tier_metro":2,
+    "src:lbl":"mz",
```
Comment (stepps00) [inline data/110/891/618/3/1108916183.geojson]: `src:lbl:centroid`?

File: data/110/891/618/7/1108916187.geojson [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "wof:belongsto":[],
+    "wof:breaches":[],
+    "wof:country":"",
+    "wof:created":1491286454,
+    "wof:geomhash":"f299eaca2c1cdebbcb129117cf614184",
+    "wof:hierarchy":[],
+    "wof:id":1108916187,
+    "wof:lastmodified":1491286454,
+    "wof:name":"Smithfield Estates",
+    "wof:placetype":"neighbourhood",
```
Comment (stepps00) [inline data/110/891/618/7/1108916187.geojson]: I thought the `"wof:repo":"whosonfirst-data",` property was machine-generated, but I guess it is not always included..

File: data/110/891/618/3/1108916183.geojson [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "wof:belongsto":[],
+    "wof:breaches":[],
+    "wof:country":"",
```
Comment (stepps00) [inline data/110/891/618/3/1108916183.geojson]: Needs a value of here, as well as an `iso:country` property, though those properties may be added during the PIP process.

**Comment by nvkelso** [comment]
Closing in favor of #722.

**Pull request closed without merging**